app/src/main/res/values/strings.xml

```diff
@@ -251,8 +251,8 @@
     <string name="grant_permission">Grant permission</string>
     <string name="clear_clp_db_confirm">Clipboard database would be completely cleared, including pinned items. Proceed?</string>
     <string name="expand_keypress_area">Expand keypress area to the edge</string>
-    <string name="update_table">Update table</string>
-    <string name="table_dict_replace_message">Table dictionary \"%1$s\" exists. You may import a new dict and replace it.</string>
+    <string name="update_table">Update character mapping table</string>
+    <string name="table_dict_replace_message">Dictionary of table-based input methods \"%1$s\" exists. You may import a new dict and replace it.</string>
```

Comment:
Those two strings are used in a dialog to prompt user if they want to replace the dict file of a single input method.

https://github.com/fcitx5-android/fcitx5-android/blob/10ff08f79b7cefbb2cb959e57fe49f89b2edb8b3/app/src/main/java/org/fcitx/fcitx5/android/ui/main/settings/TableInputMethodFragment.kt#L352-L355

`update_table` is used for dialog title, it shoud be "Update dictionary table-based input method". But for simplicity I'd keep "Update table". It's for one input method, so plural is unnecessary.